### PR TITLE
Changed colormap3 to support more than 2 members and changed durations

### PR DESCRIPTION
**Pull request by CuteFwan**
I know this was code copied from me, but its horrible outdated half-done code so I fixed it up and added some features.

### CODE REVIEW

File: dango/plugins/fun.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def _gifmap(avys):
 
         start = time.time()
         async with ctx.typing():
-            avys = await asyncio.gather([fetch_image(t.avatar_url_as(format="png")) for t in [source + list(dest)]])
+            avys = await asyncio.gather(*[fetch_image(t.avatar_url_as(format="png")) for t in [source] + list(dest)])
```
Comment (khazhyk) [inline dango/plugins/fun.py]: pls squash this onto whatever introduced the typo

File: dango/plugins/fun.py [hunk 2]
```diff
@@ -154,53 +154,65 @@ def __init__(self, cfg, res):
...
-
-            base = Image.new('RGBA', (maxres,maxres))
-            basedata = base.load()
-            for i, d in enumerate(avy1data):
-                x1, y1 = d[0]
-                x2, y2 = avy2data[i][0]
-                x, y = round(x1 + (x2 - x1)*m), round(y1 + (y2 - y1) * m)
-                basedata[x, y] = avy2data[i][1]
-            frames.append(base)
+        for j, im in enumerage(imgs[1:]):
```
Comment (khazhyk) [inline dango/plugins/fun.py]: whats this

File: dango/plugins/fun.py (hunk 1 above)
Comment (CuteFwan) [reply dango/plugins/fun.py]: Squashed it

**Comment by khazhyk** [comment]
I don't know why it pushed those commits (I was trying to push 816d33399e8dffdaab633595415224ca2a69f3db to your repo but it instead pushed... something else entirely. odd). Anyways, thanks, merged

**Pull request closed without merging**